### PR TITLE
Create a new example with Electron remote

### PR DESCRIPTION
**Pull request by SebastienWae**
## This PR contains:
- NEW EXAMPLE

## Describe the problem you have without this PR
None.
This is another example of implementing RxDB with Electron but using `remote` instead of `RxDB server plugin`.

### CODE REVIEW

File: examples/electron-remote/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# RxDB Electron Remote example
+
+This is an example usage of RxDB with Electron and `remote` instead of the `RxDB server plugin`. It implements a simple heroes-list which can be filled by the user.
```
Comment (pubkey) [inline examples/electron-remote/README.md]: I think this should link to a description what 'remote' means with electron.
https://electronjs.org/docs/api/remote

**Comment by pubkey** [comment]
Looks good, I added one comment and I restarted the CI

File: examples/electron-remote/README.md (hunk 1 above)
Comment (SebastienWae) [reply examples/electron-remote/README.md]: Done, I also added links to the other example in each of the Electron README files.

**Comment by pubkey** [comment]
Thank you for that great PR. I merged it

**Pull request merged**